Product/Production/Services/DocumentQueryCore/src/main/java/gov/hhs/fha/nhinc/docquery/audit/transform/DocQueryAuditTransforms.java

```diff
@@ -215,18 +211,17 @@ private ParticipantObjectIdentificationType createPatientParticipantObjectIdenti
     }
 
     private ParticipantObjectIdentificationType createParticipantObject(short objTypeCodeSys, short objTypeCodeRole,
-        String objIdTypeCode, String objIdTypeCodeSys, String objIdTypeDisplayName) {
+            String objIdTypeCode, String objIdTypeCodeSys, String objIdTypeDisplayName) {
 
         return createParticipantObjectIdentification(objTypeCodeSys, objTypeCodeRole,
-            objIdTypeCode, objIdTypeCodeSys, objIdTypeDisplayName);
+                objIdTypeCode, objIdTypeCodeSys, objIdTypeDisplayName);
     }
 
     private byte[] getParticipantObjectQueryForRequest(AdhocQueryRequest request) throws JAXBException {
         StringWriter sw = new StringWriter();
```

Comment:
We use a ByteArrayOutputStream for other services (e.g., PD) -- any reason we're using a StringWriter here?